src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -784,6 +810,11 @@ void cfg_ecmd2xcmd(struct ert_configure_cmd *ecmd,
 {
 	int i;
 
+	/* To let ERT 3.0 firmware aware new KDS is talking to it,
+	 * set kds_30 bit. This is no harm for ERT 2.0 firmware.
+	 */
+	ecmd->kds_30 = 1;
```

Comment:
Looking at the comment suggest that this value is a version number or is this a boolean value.  Sorry, I am note clear on the intent.

Reply:
This define as just 1 bit in ert.h.
"  uint32_t kds_30:1;"